sql/core/src/main/java/org/apache/spark/sql/execution/vectorized/WritableColumnVector.java

```diff
@@ -72,7 +72,7 @@ public void reset() {
       numNulls = 0;
     }
 
-    if (hugeVectorThreshold > 0 && capacity > hugeVectorThreshold) {
```

Comment:
`= 0` means we treat all vectors as huge vectors?

Reply:
Yes， but  `= 0` is not recommended, just keep code and documentation consistent

Reply:
then shall we update the doc instead?

Reply:
Changing the doc will result in code part change as other places test if hugeVectorThreshold < 0

Reply:
ah I see